src/mat2d.js

```diff
@@ -7,16 +7,16 @@ import * as glMatrix from "./common.js";
  * @description
  * A mat2d contains six elements defined as:
  * <pre>
- * [a, c, tx,
- *  b, d, ty]
+ * [a, b, c,
+ *  d, tx, ty]
```

Comment:
This should probably be written as
```
[a, b,
 c, d,
 tx, ty]
```

My brain initially read it the wrong way because of the shape.

Reply:
Good point. I fixed it in https://github.com/toji/gl-matrix/commit/120fe1057b510d4a49c3bcaffeb2b8cd52dd0cc0